nibe/data/s1155_s1255.csv

```diff
@@ -1131,13 +1144,13 @@ Max difference, SES lower priority energy source	MODBUS_HOLDING_REGISTER	1327	10
 Delta brine pump passive cooling	MODBUS_HOLDING_REGISTER	1328	1	°C	4	1	15	4
 Installed (EB101)	MODBUS_HOLDING_REGISTER	1550	1		4	0	1	0
 EME20	MODBUS_HOLDING_REGISTER	4000	1		4	0	1	0
-Current power (EME 20)	MODBUS_INPUT_REGISTER	2176	1	W	6	0	0	0
+Current power	MODBUS_INPUT_REGISTER	2176	1	W	6	0	0	0
 Speed of circulation pump passive cooling (EP15)	MODBUS_HOLDING_REGISTER	4006	1	%	4	1	100	70
 Speed of circulation pump active cooling (EP15)	MODBUS_HOLDING_REGISTER	4007	1	%	4	1	100	70
 Ground water pump's control signal	MODBUS_HOLDING_REGISTER	4012	1		1	0	0	0
 BT12 offset	MODBUS_HOLDING_REGISTER	4014	10	°C	1	-50	50	0
-Total average power (EME 20)	MODBUS_INPUT_REGISTER	2178	1	W	6	0	0	0
-Total energy (EME 20)	MODBUS_INPUT_REGISTER	2180	10	kWh	3	0	0	0
+Total average power (EME 20)	MODBUS_INPUT_REGISTER	2178	100	kW	6	0	0	0
```

Comment:
Factor 100 seems to be too high. Interesting if it is correct.

Reply:
Seen same for F series. So i think its correct.